vhdl_parser/src/ast.rs

```diff
@@ -972,6 +995,7 @@ pub struct ConfigurationDeclaration {
     pub ident: Ident,
     pub entity_name: SelectedName,
     pub decl: Vec<ConfigurationDeclarativeItem>,
+    pub vunit_bind_inds: Option<Vec<VUnitBindingIndication>>,
```

Comment:
Maybe Option<Vec> could just be a Vec. None would be an empty Vec.

Reply:
Ok, makes sense. I'll remove it.